sdk/spring/azure-spring-boot-starter-storage/README.md

```diff
@@ -72,6 +72,21 @@ private final BlobServiceAsyncClient blobServiceAsyncClient = blobServiceClientB
 
 ```
 
+#### Search for resources
+You can use implementation class `AzureStorageResourcePatternResolver` of `ResourcePatternResolver` to search resource, it supports `blob` or `file` type.
+* Pattern search, the **searchPattern** should start with `azure-blob://` or `azure-file://`. Such as `azure-blob://*/*`, it means list all blobs in all containers; `azure-blob://demo-container/**`, it means list all blobs in the demo-container container, including any sub-folder.
+* Location search, the **searchLocation** should start with `azure-blob://` or `azure-file://`, the remaining file path should exist, otherwise an exception will be thrown.
+
+```java
+AzureStorageResourcePatternResolver storageResourcePatternResolver = new AzureStorageResourcePatternResolver(blobServiceClientBuilder.buildClient());
+
+Resource[] resources = storageResourcePatternResolver.getResources(searchPattern);
+Resource resource = storageResourcePatternResolver.getResource(searchLocation);
+```
+
+#### Multipart upload
+Files larger than 4 MiB will be uploaded to Azure Storage in parallel.
```

Comment:
nit: any sources of this 4 MiB?

Reply:
Keep it consistent with the official website.